README.md

```diff
@@ -64,6 +66,16 @@ class MyDaemonizedCommand extends DaemonCommand
 
         // Add your own optional callback : called every 10 iterations
         $this->addIterationsIntervalCallback(10, [$this, 'executeEveryTenLoops']);
+        
+        // You can add your own Periodic Timer,
+        // In this timer  will be called every 0.5s.
+        $daemon = $this;
```

Comment:
i'm pretty sure that PHP already binds `$this` with its parent context when creating a anonymous function

So i'm 90% sure that you can simply do:

```php
$this->loop->addPeriodicTimer(0.5, function ($timer) {
    if ($this->isLastLoop()) {
        $this->loop->cancelTimer($timer);
    }
});
```

Reply:
That is true... Since PHP 5.4 :P [See here for details](http://php.net/manual/fr/functions.anonymous.php#functions.anonymous-functions)

:+1:  Thank you

Reply:
thanks for the link  ❤️